### ISSUE TITLE
graylog-plugin-aws 0.6.0 not working on new instance of Graylog 2.0.3

### ISSUE THREAD

**Issue by stepkirk** [Graylog2/graylog-plugin-aws#12]
This is strange since the plugin IS working on another instance of Graylog 2.0.3 that I upgraded from a Graylog 1.3.4 instance.

BTW, this is the same error that @123dev reported in issue #9 after it was closed.  Didn't see a response over there so opened this new issue.

This is the error:

```
2016-07-14_17:06:19.87439 2016-07-14 11:06:19,874 INFO : com.graylog2.input.cloudtrail.CloudTrailTransport - Starting cloud trail subscriber
2016-07-14_17:06:19.87516 2016-07-14 11:06:19,874 INFO : org.graylog2.inputs.InputStateListener - Input [AWS CloudTrail Input/5787c68b4335af0398c2fd03] is now STARTING
2016-07-14_17:06:19.87725 Exception in thread "Thread-16" java.lang.NoClassDefFoundError: Could not initialize class com.amazonaws.http.conn.ssl.SdkTLSSocketFactory
2016-07-14_17:06:19.87827       at com.amazonaws.http.apache.client.impl.ApacheConnectionManagerFactory.getPreferredSocketFactory(ApacheConnectionManagerFactory.java:87)
2016-07-14_17:06:19.87958       at com.amazonaws.http.apache.client.impl.ApacheConnectionManagerFactory.create(ApacheConnectionManagerFactory.java:65)
2016-07-14_17:06:19.88028       at com.amazonaws.http.apache.client.impl.ApacheConnectionManagerFactory.create(ApacheConnectionManagerFactory.java:58)
2016-07-14_17:06:19.88089       at com.amazonaws.http.apache.client.impl.ApacheHttpClientFactory.create(ApacheHttpClientFactory.java:50)
2016-07-14_17:06:19.88136       at com.amazonaws.http.apache.client.impl.ApacheHttpClientFactory.create(ApacheHttpClientFactory.java:38)
2016-07-14_17:06:19.88238       at com.amazonaws.http.AmazonHttpClient.<init>(AmazonHttpClient.java:259)
2016-07-14_17:06:19.88286       at com.amazonaws.AmazonWebServiceClient.<init>(AmazonWebServiceClient.java:145)
2016-07-14_17:06:19.88360       at com.amazonaws.AmazonWebServiceClient.<init>(AmazonWebServiceClient.java:136)
2016-07-14_17:06:19.88405       at com.amazonaws.AmazonWebServiceClient.<init>(AmazonWebServiceClient.java:121)
2016-07-14_17:06:19.88493       at com.amazonaws.services.sqs.AmazonSQSClient.<init>(AmazonSQSClient.java:229)
2016-07-14_17:06:19.88546       at com.amazonaws.services.sqs.AmazonSQSClient.<init>(AmazonSQSClient.java:209)
2016-07-14_17:06:19.88640       at com.graylog2.input.cloudtrail.notifications.CloudtrailSQSClient.<init>(CloudtrailSQSClient.java:39)
2016-07-14_17:06:19.88734       at com.graylog2.input.cloudtrail.CloudTrailSubscriber.run(CloudTrailSubscriber.java:63)
2016-07-14_17:06:19.88764 2016-07-14 11:06:19,877 INFO : org.graylog2.inputs.InputStateListener - Input [AWS CloudTrail Input/5787c68b4335af0398c2fd03] is now RUNNING
```

What am I missing in this new instance?  What else can I try to troubleshoot?

FWIW, I tried deleting and re-adding the Input with the same result.

Thanks,
Steve.

**Comment by joschi** [comment]
@stepkirk How did you download and install the plugin?

Because the class in question (`com.amazonaws.http.conn.ssl.SdkTLSSocketFactory`) is definitely part of the plugin JAR file:
```
$ unzip -l graylog-plugin-aws-0.6.0.jar | grep SdkTLSSocketFactory
     7928  07-07-16 12:44   com/amazonaws/http/conn/ssl/SdkTLSSocketFactory.class
$ md5sum graylog-plugin-aws-0.6.0.jar
b5e0e8655a9fc1ec3e85102db9187aba  graylog-plugin-aws-0.6.0.jar
```

**Comment by 123dev** [comment]
In Our case, 
```
$ md5sum graylog-plugin-aws-0.6.0.jar
b5e0e8655a9fc1ec3e85102db9187aba  graylog-plugin-aws-0.6.0.jar

$ unzip -l graylog-plugin-aws-0.6.0.jar | grep SdkTLSSocketFactory
     7928  2016-07-07 12:44   com/amazonaws/http/conn/ssl/SdkTLSSocketFactory.class
```

So it's not about bad download, at least I don't think so.
Thanks

**Comment by stepkirk** [comment]
FWIW, my output matches yours @joschi and @123dev.

Anything else I can check?

Thanks,
Steve.

**Comment by stepkirk** [comment]
I seem to be having the same experience @123dev was having over in issue #9 .  
I tried to re-download the 0.6.0 version of the plugin but (as expected) it made no difference.
I then pulled the 0.5.1 version linked to in issue #10 and that version is working, just as @123dev had discovered.

So, on one Graylog 2.0.3 instance I am able to get the AWS Cloudtrail logs using version 0.6.0 of the plugin.  On the other/newer Graylog 2.0.3 instance, I have to use version 0.5.1 of the plugin to get the Cloudtrail logs.

Not sure what this might indicate but I wanted to pass along the information.
